app/controllers/hyrax/batch_ingest/batches_controller.rb

```diff
@@ -65,6 +64,10 @@ def available_ingest_types
         def batch_params
           params.require(:batch).permit(:submitter_email, :ingest_type, :admin_set_id)
         end
+
+        def start_batch_runner(batch)
+          Hyrax::BatchIngest::BatchRunner.new(batch: batch).initialize_batch
```

Comment:
I think that we may want this to be `#run` instead of `#initialize_batch` so the batch will get all of the way to `enqueued`.

Reply:
Cool. I'll have to make a call to `BatchRunner#run` first, and then check `@batch.valid?` like you suggested